lib/src/constants/constants.dart

```diff
@@ -0,0 +1,23 @@
+import 'package:flutter/material.dart';
+
+abstract class Constants {
+  static const double containerPaddingRightLeft = 10.0;
+  static const double containerPaddingTop = 25.0;
+  static const double safeAreaPaddingVertical = 73.0;
+  static const double safeAreaPaddingHorizontal = 20.0;
+  static const double borderWidth = 2.0;
+  static const double circleAvatarRadius = 120.0;
+  static const double gradientRadius = 0.65;
+  static const double paddingPosition = 10.0;
+  static const double fontSize = 30.0;
+  static const double cardFontSize = 15.0;
+  static const double principalContainerHeight = 700.0;
+  static const double principalContainerWidth = 350.0;
+  static const double borderRadius = 10.0;
+  static const Color cardTextColor = Colors.yellowAccent;
```

Comment:
This const is not necessary because the color itself is a constant too

Reply:
Fixed!